source/Magritte-Model/Symbol.extension.st

```diff
@@ -5,11 +5,6 @@ Symbol >> asAccessor [
 	^ MASelectorAccessor selector: self
 ]
 
-{ #category : #'*Magritte-Model' }
-Symbol >> cull: receiverObject cull: argumentObject [ 
```

Comment:
This seems really more "not-bloc" than "not-GT", but I guess maybe it doesn't matter for now. A simpler long-term fix would be to include Symbol>>#cull:cull: in Pharo